extensions/rich_text_components/Image/directives/ImageDirective.js

```diff
@@ -35,10 +35,22 @@ oppia.directive('oppiaNoninteractiveImage', [
         $scope.filepath = HtmlEscaperService.escapedJsonToObj(
           $attrs.filepathWithValue);
         $scope.imageUrl = '';
-        ImagePreloaderService.getImageUrl($scope.filepath)
-          .then(function(objectUrl) {
+
+        if (ImagePreloaderService.inExplorationPlayer()) {
+          ImagePreloaderService.getImageUrl($scope.filepath
+          ).then(function(objectUrl) {
+            $scope.imageUrl = objectUrl;
+          });
```

Comment:
In this file just call `ImagePreloaderService.getImageUrl(` as before.

Reply:
Why ?
We need to check whether we are playing an exploration or editing it. If we are in exploration player then the images are in the process of loading (ImagePreloaderService) but in case of editing the exploration, the images are not loaded because the service has not been called yet. We need to display the image for the preview, hence we need to check which mode is the exploration in.

Reply:
Done!